ui/src/plugins/logger/Logger.js

```diff
@@ -0,0 +1,49 @@
+export default () => {
+  const isDebugEnabled = process.env.DEBUGGING
+
+  const normalizeMessage = msg => `%c ${msg}`
+  const getStyle = isError => (
+    `
+      background: ${isError ? '#C10015 ' : '#1976d2'};
+      font-weight: bold;
+      color: white;
+      padding: 6px 20px;
+      border-radius: 4px;
+      display: block;
+      width: 100%;
+      font-size: 12px;
+    `
+  )
+
+  return {
+    group (message, payload = [], isErrorType) {
```

Comment:
Aqui o isErrorType não pode assumir o default `false`?

Reply:
Quando não passo nada como está, ele já é um valor `falsy`, no caso undefined, acha que vê necessidade de forçar um boolean?

Reply:
Considerando que não fazemos isto em nenhum outro lugar, para saber o tipo, vou manter assim, faria muito sentido desta forma utilizando typescript.